### PR TITLE
Update to version 9.54.0

### PR DESCRIPTION
**Pull request by mbargull**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

supersedes / closes gh-19

Trying without added patches from gh-19.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#         _cups_close in gdevcups.o
+#     "_cupsRasterOpen", referenced from:
+#         _cups_print_pages in gdevcups.o
+#     "_cupsRasterWriteHeader2", referenced from:
+#         _cups_print_pages in gdevcups.o
+#     "_cupsRasterWritePixels", referenced from:
+#         _cups_print_pages in gdevcups.o
+#         _cups_print_chunked in gdevcups.o
+#   ld: symbol(s) not found for architecture x86_64
+#   clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
```
Comment (mbargull) [inline recipe/conda_build_config.yaml:31]: @isuruf, did you ever encounter a case such like this one here?
The linker doesn't find symbols from `libcupsimage` (which is in `./usr/lib/` in the downloaded SDK), but only for `clang=11`; `clang=10` works fine.
Ghostscript uses some indirections in its build, vendors things etc. which is why I can't easily see the cause and don't have time to look further. And, of course, I wouldn't want to get you to do any of that -- I just wanted to ask if you might have a hunch on possible culprits.

The rest of the build log is here:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=315819&view=logs&j=58ac6aab-c4bc-5de2-2894-98e408cc8ec9&t=933f325c-924e-533d-4d95-e93b5843ce8b&l=10741

Comment (mbargull) [reply recipe/conda_build_config.yaml:31]: (cc @FelixMoelder)

Comment (isuruf) [reply recipe/conda_build_config.yaml:31]: Looks like with clang-11, `libcupsimage` is not linked in to the binary and with clang-10, `cups` support is disabled. Not sure why

Comment (mbargull) [reply recipe/conda_build_config.yaml:31]: Thanks Isuru, that makes sense. (Plus I'm facepalming that I didn't inspect the clang-10 build logs more closely >_< .)

**Pull request merged**